addons/website_sale_picking/models/website.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    picking_site_ids = fields.Many2many('delivery.carrier', string='Picking sites',
+                                        compute='_compute_picking_sites', store=False)
```

Comment:
`store=False` is implicit here.
```suggestion
    picking_site_ids = fields.Many2many('delivery.carrier', string='Picking sites',
                                        compute='_compute_picking_sites')
```